ansible/roles/debops.docker/defaults/main.yml

```diff
@@ -0,0 +1,391 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# debops.docker_client default variables
+# ======================================
+
+# .. contents:: Sections
+#    :local:
+#
+# .. include:: ../../../includes/global.rst
+
+
+# APT packages [[[
+# ------------------------------
```

Comment:
The underline should be flush with the section title. It's not a bug in the Sphinx parser, but it looks better in the source code.